server.js

```diff
@@ -1,299 +1,299 @@
-
-const PORT = 80;
-const HOSTNAME = '0.0.0.0';
-
-const uri = "mongodb+srv://suraj_admin:suraj_admin@cluster0.dme40pl.mongodb.net/?retryWrites=true&w=majority";
-
-async function startServer() {
-  try {
-    // Connect to MongoDB using Mongoose
-    await mongoose.connect(uri, { useNewUrlParser: true, useUnifiedTopology: true });
-    console.log('Connected to MongoDB');
-
-    // Create HTTP Server
-    const server = http.createServer((req, res) => {
-      res.statusCode = 200;
-      res.setHeader('Content-Type', 'text/plain');
-      res.end('Hello World\n');
-    });
-
-    // Start HTTP Server
-    server.listen(PORT, HOSTNAME, () => {
-      console.log(`Server running at http://${HOSTNAME}:${PORT}/`);
-    });
-  } catch (err) {
-    console.error('Could not connect to MongoDB:', err);
-    process.exit(1);
-  }
-}
-
-startServer();
-
- */
-
-
-
-//TASK 4:Create Document for slam book
-/*
-const slambookSchema = new mongoose.Schema({
-  nameInYourContact: String,
-  relationship: String,
-  somethingYouLikeInMe: String,
-  somethingYouHateInMe: String,
-  ifIDieYourReaction: String,
-  whatDidYouFeelWhenYouFirstSawMe: String,
-  beutifulMessageForMe: String,
-  nickNameForMe: String,
-  songDedicatedToMe: String,
-  canIShare: String,
-  yourName: String
-});
-
-const SlamBook = mongoose.model('SlamBook', slambookSchema);
-*/
-
-
-
-//TASK5: Write sample crud endpoints
-/**
-const http = require('http');
-const url = require('url');
-const mongoose = require('mongoose');
-const { StringDecoder } = require('string_decoder');
-
-const PORT = 80;
-const HOSTNAME = '0.0.0.0';
-
-const server = http.createServer((req, res) => {
-  const parsedUrl = url.parse(req.url, true);
-  const path = parsedUrl.pathname.replace(/^\/+|\/+$/g, '');
-  const method = req.method.toLowerCase();
-
-  // Buffer the data
-  const decoder = new StringDecoder('utf-8');
-  let buffer = '';
-  req.on('data', (data) => {
-    buffer += decoder.write(data);
-  });
-
-  req.on('end', () => {
-    buffer += decoder.end();
-
-    let responseContent = 'Not Found\n';
-    let statusCode = 404;
-
-    // Sample CRUD Endpoints
-    if (path === '') {
-      switch (method) {
-        case 'get':
-          responseContent = 'Read: Sample Data\n';
-          statusCode = 200;
-          break;
-        case 'post':
-          responseContent = 'Create: Data Received - ' + buffer + '\n';
-          statusCode = 201;
-          break;
-        case 'put':
-          responseContent = 'Update: Data Received - ' + buffer + '\n';
-          statusCode = 200;
-          break;
-        case 'delete':
-          responseContent = 'Delete: Sample Data\n';
-          statusCode = 200;
-          break;
-      }
-    }
-
-    res.writeHead(statusCode, { 'Content-Type': 'text/plain' });
-    res.end(responseContent);
-  });
-});
-
-server.listen(PORT, HOSTNAME, () => {
-  console.log(`Server running at http://${HOSTNAME}:${PORT}/`);
-});
- */
-
-
```

Comment:
The code for Task 6-9 is commented out. Please remove the commented code to ensure the code can be executed properly.